src/components/bootcamps/BootcampCreatePage.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import * as yup from 'yup'
+import { useRouter } from 'next/router'
+import { FormikHelpers } from 'formik'
+import { useMutation } from 'react-query'
+import { useTranslation } from 'next-i18next'
+import { AxiosError, AxiosResponse } from 'axios'
+import { Container, Grid, Typography } from '@mui/material'
+
+import { routes } from 'common/routes'
+import { createBootcamp } from 'common/rest'
+import { AlertStore } from 'stores/AlertStore'
+import GenericForm from 'components/common/form/GenericForm'
+import SubmitButton from 'components/common/form/SubmitButton'
+import FormTextField from 'components/common/form/FormTextField'
+import { ApiErrors, isAxiosError, matchValidator } from 'common/api-errors'
+import { BootcampResponse, BootcampFormData, BootcampInput } from 'gql/bootcamps'
+import Layout from './layout/Layout'
+
+const validationSchema: yup.SchemaOf<BootcampFormData> = yup
+  .object()
+  .defined()
+  .shape({
+    firstName: yup.string().trim().min(3).max(25).required(),
+    lastName: yup.string().trim().min(3).max(25).required(),
+  })
+
+const defaults: BootcampFormData = {
+  firstName: '',
+  lastName: '',
+}
+
+export type BootcampFormProps = { initialValues?: BootcampFormData }
+
+export default function BootcampForm({ initialValues = defaults }: BootcampFormProps) {
+  const { t } = useTranslation()
```

Comment:
If you want to use specific translation namespace you can pass it to the `useTranslation` hook as:

```tsx
const { t } = useTranslation('bootcamps')

// then
t('bootcamps:bootcamp.firstName')
// becomes
t('bootcamp.firstName')
```